src/main/com/intellij/lang/jsgraphql/ide/references/GraphQLReferenceService.java

```diff
@@ -230,6 +230,19 @@ public void visitElement(final @NotNull PsiElement schemaElement) {
                     }
                 });
             }
+            if (name.startsWith("_")) {
```

Comment:
One more thing, why did you add it here? AFAIU, those fields are added only on the root level, so it should work with the default reference resolution mechanism. Did you face any issues with it? Those fields are declared in the Query type extension, it should work out of the box:
```
extend type Query {
  _entities(representations: [_Any!]!): [_Entity]!
  _service: _Service!
}
```

Reply:
I wrote a test https://github.com/jimkyndemeyer/js-graphql-intellij-plugin/blob/61d1694f532ade59e354f36764bd895c6a54388a/src/test/com/intellij/lang/jsgraphql/frameworks/federation/GraphQLFederationResolveTest.java#L33 and removed those lines. If you think they are necessary pls provide a test case for it, I can't find one myself.

Reply:
Sounds good. I agree, it's not required.